group_project/assignment_3.py

```diff
@@ -1,7 +1,10 @@
 def replace_bad_words(a_string, word_to_replace, new_word):
-    pass
-
-
+    if word_to_replace in a_string:
+        a_string = a_string.replace(word_to_replace, new_word)
+        return a_string
+    return a_string
+    
```

Comment:
Good job. You did not put a redundant elif like I did. If you haven't, check out Santiagos solution to this in his email. This can be done without an if statement.